tests/scans/code_analysis/README.md

```diff
@@ -1,63 +1,55 @@
-  - We move the vulnerabilities to the `to_fix` key of the known flaws JSON file.
-  - If the new vulnerability is considered a false positive, we add it to the `false_positives` list of the dictionary in its respective `known_flaws` JSON file.
-  - If the new vulnerability is a real vulnerability, we solve the problem reported and remove the flaw from the known flaws file.
+- `--repo`: the repository used. 
 
-The test also updates the known_flaws files automatically. If we have a look at a known_flaws file, we will see that each flaw dictionary contains information like the line number or range. This information is the one updated by the test. The test also removes flaws from the known_flaws file if they don't appear in the Bandit output.
+    Default: `wazuh`
+- `--reference`: the repository branch. 
 
-#### Parameters
+    Default: `master`
+- `--check_directories`: the directories names to check. This parameter accepts multiple directories separated by commas.
 
-As said in the description, the test uses `Bandit` to look for possible Python flaws. By default, the tests checks the framework, wodles and api directories in the Wazuh repository, in its master branch.
+    Default: `framework/,api/,wodles/`
+- `--exclude_directories`: the directories names to exclude. This parameter accepts multiple directories separated by commas. 
 
-These directories, repository and branch can be passed to the test as parameters so it is possible to run the test in any directory containing Python code inside the Wazuh organization.
+    Default: `test/,tests/`
+- `--confidence`: the minimum value of confidence of the Bandit scan. This value must be 'UNDEFINED', 'LOW', 'MEDIUM' or 'HIGH'.
 
-Apart from this parameters, there are more that can be used to customize the test functionality. Note that the test will fail if we check different directories and/or repository as we don't have known_flaws files for non-default directories.
+    Default: `MEDIUM`
+- `--severity`: the minimum value of severity of the Bandit scan. This value must be 'UNDEFINED', 'LOW', 'MEDIUM' or 'HIGH'.
 
-- **--repo**: set the repository used. Default: `wazuh`
-- **--reference**: set the repository branch or tag. Default: `master`
-- **--check_directories**: set the directories to check, this must be a string with the directory name.
-If more than one is indicated, they must be separated with comma. Default: `framework/,api/,wodles/`.
-- **--exclude_directories**: set the directories to exclude, this must be a string with the directory name.
-If more than one is indicated, they must be separated with comma. Default: `test/,tests/`.
-- **--confidence**: set the minimum value of confidence of the Bandit scan.
-This value must be 'UNDEFINED', 'LOW', 'MEDIUM' or 'HIGH'. Default: `MEDIUM`
-- **--severity**: set the minimum value of severity of the Bandit scan.
-This value must be 'UNDEFINED', 'LOW', 'MEDIUM' or 'HIGH'. Default: `LOW`
+    Default: `LOW`
```

Comment:
The README uses backticks and quotes for the variable values. I suggest to use always backticks
For example, replace 'UNDEFINED' by `UNDEFINED`